b2share/modules/communities/models.py

```diff
@@ -182,12 +185,15 @@ def create_community_oaiset(community, fix=False):
     :param fix: Enable the fixing of the database. This function doesn't fail
         if the oai set already exists. It will just add it if it is missing.
     """
+
+    from b2share.utils import add_to_db
+
```

Comment:
Why import multiple times in different methods as opposed to importing at the start?